cloudify_aws/rds/tests/test_instance_read_replica.py

```diff
@@ -94,7 +97,7 @@ def test_create_raises_UnknownServiceError(self):
             )
 
         self.assertEqual(
-            str(error.exception),
+            text_type(error.exception),
```

Comment:
Same question. No change is necessary, just curious.

Reply:
Same as above